low-level/files/opt/seagate/sspl/conf/sspl.conf.LR2.yaml

```diff
@@ -180,6 +180,23 @@ SYSTEMDWATCHDOG:
    monitor: true
    threaded: true
    monitored_services:
+      - hare-consul-agent.service
+      - hare-consul-agent-c1.service
```

Comment:
I think hare-consul-agent-c1 and c2 won;t be there in LR R2.

Reply:
thanks @madhavemuri 
@mssawant @ajaysrivas  can you confirm please, do we need to have these in monitoring list?